v03_pipeline/lib/tasks/base/base_variant_annotations_table.py

```diff
@@ -21,12 +23,11 @@ def output(self) -> luigi.Target:
 
     def requires(self) -> list[luigi.Task]:
         return [
-            HailTableTask(
-                valid_reference_dataset_collection_path(
-                    self.reference_genome,
-                    self.dataset_type,
-                    rdc,
-                ),
+            UpdatedReferenceDatasetCollectionTask(
```

Comment:
Yep this look like the right start to me.  I'd grep for more though!

Reply:
talked offline. I found other references to `HailTableTask` but they're for the cached query reference dataset, for which we have a refactor planned later as part of this epic.